php/public/talkjs-config.js

```diff
@@ -0,0 +1,7 @@
+import Talk from 'talkjs';
+
+const talkjs = new Talk({
```

Comment:
Is this actually used? I see that you're putting in the app ID by hand in chat.blade.php, which seems fine to me. If that's the only place you're using it, this file can be deleted

Reply:
yes , it can be deleted